### ISSUE TITLE
EOF is not part of shanghai

### ISSUE THREAD

**Issue by holiman** [ethereum/tests#1195]
See e.g. https://github.com/ethereum/tests/blob/develop/BlockchainTests/GeneralStateTests/EIPTests/stEOF/ori/CreateEOF1.json

This test assumes that EOF is part of shanghai, which is not the case. This causes thousands of hive-errors. 

Suggested solution: redefine them as either `Cancun` or some other `Fork+Eip` combo.

**Comment by axic** [comment]
@hugo-dc last week has been working on changing every EOF test to Cancun and refilling them. It should be done soon?

First tranche is #1193.

**Comment by winsvega** [comment]
The EIPTests folder is about experimental tests. We move it now to a separate folder so not to confuse. This are not mandatory tests. we put a README in the folder. but still it is causing confusion. so all experimental tests (not updated with specs or tests for upcoming EIPS are now in EIPTests folder  /src/EIPTestsFiller )
https://github.com/ethereum/tests/pull/1192

@holiman  any objections?

**Comment by holiman** [comment]
Ah, ok, once it's settled which folder to ignore (the `BlockchainTests/GeneralStateTests/EIPTests/` or `/EIPTests`), ping me and we can make sure that hive also ignores it.

**Comment by winsvega** [comment]
tests/EIPTests

**Comment by winsvega** [comment]
now we don't care if it is Shanghai or not. since the meaning of Shanghai in EIPTests is `Shanghai+EIPXXXX`  
the decs who generate tests on custom fork now it. the test hash commit hash. but the dev still has to provide the src of geth where this commit hash comes from (to be able to regenerate the test) 

then once the EIP is merged into fork and the tests pass. and we are clear about the specs. 
we move tests from EIPTests folder into  `/tests/GeneralStateTests/Cancun` and bc folder accordingly 
and that will be required to run by clients and on hive
I also keep track how clients provided t8n's are passing the tests. 
I have  ethereumjs, geth, besu, nimbus now
